packages/job-worker/src/playout/model/PlayoutModel.ts

```diff
@@ -342,6 +342,8 @@ export interface PlayoutModel extends PlayoutModelReadonly, StudioPlayoutModelBa
 	 */
 	setQuickLoopMarker(type: 'start' | 'end', marker: QuickLoopMarker | null): void
 
+	getSegmentsBetweenQuickLoopMarker(start: QuickLoopMarker, end: QuickLoopMarker): SegmentId[]
```

Comment:
it would be good to have some documentation on this method. almost every other method on this interface does